src/Umbraco.Web/Search/UmbracoIndexesCreator.cs

```diff
@@ -80,28 +84,31 @@ private IIndex CreateMemberIndex()
                 CreateFileSystemLuceneDirectory(Constants.UmbracoIndexes.MembersIndexPath),
                 new CultureInvariantWhitespaceAnalyzer(),
                 ProfilingLogger,
-                GetMemberValueSetValidator());
+                GetMemberValueSetValidator()
+                );
             return index;
         }
-        
+        [Obsolete]
```

Comment:
We need to have a description of why this is obsoleted in the ctor overload of this attribute, same goes for the other obsoleted ones.